tests/snmpsim/ceraos.snmprec

```diff
@@ -1,2 +1 @@
-1.3.6.1.2.1.1.1.0|4|High capacity packet radio outdoor unit
-1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281
```

Comment:
This needs to stay as it was, the test is supposed to simulate a device and not the bare minimum the code is checking for.